torch/_meta_registrations.py

```diff
@@ -51,6 +53,24 @@ def register(op):
     return wrapper
 
 
+def register_meta_foreach(op):
+    def wrapper(fn):
+        fn = _convert_out_params(fn)
+
+        def register(op):
+            scalar_op_name, _ = str(op).replace("aten._foreach_", "").split(".")
+            scalar_op = getattr(aten, scalar_op_name).default
+            scalar_fn = decomposition_table.get(scalar_op, None)
+            if not scalar_fn:
+                scalar_fn = meta_table[scalar_op]
```

Comment:
Isn't it enough to use the `scalar_op` here?

Reply:
Yes, fixed now.